packages/code-du-travail-data/dataset/code_du_travail/eposeidon_script/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "build_eposeidon",
+  "name": "@cdt/data...build_eposeidon",
```

Comment:
why there is `...` in packages name ?

Reply:
why not :) 

- `/` si prohibited so we need to find something else as separator
- `-` and `_` are too common

Reply:
I made an article about `...` back then :)
https://medium.com/@douglasduteil/npm-scoping-for-free-f162c1db4638

Reply:
> - and _ are too common
😭 I like - and _ 

On a more general thinking I thought you would move those packages to the monorepo root.

Reply:
Maybe under an `@cdt/dataset` namespace 
however moving the files will alter the Python indexation script.
@revolunet do we have kind of Python tests that can help us knowing if the indexation script still work after moving the dataset files ?

Reply:
the create_indexes script itself should fail when it cannot find source files